pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -377,6 +378,23 @@ func podSpec(containers []corev1.Container, opts ...podSpecOption) *corev1.PodSp
 	return podSpec
 }
 
+func withAppendedTokenVolumes(filenames []string, audiences []string, expiries []*int64) podSpecOption {
```

Comment:
nit: `expiries` looks like a typo, is there another name we could use?

Reply:
expires